### DISCUSSION TITLE
Is there a way to reconfigure the dhcp plugin without restarting strongswan?

### DISCUSSION THREAD

**Discussion by HolyDLX** [strongswan/strongswan#913] [Q&A] [answered]
Hi,

I am new to the strongswan topic and was looking into a setup I came across.
Part of the setup is a configuration of the dhcp plugin.

Is there a way to reconfigure the plugin (e.g. change Server) without restarting strongswan?
Maybe I am overlooking something.

Kind regards

**Comment by tobiasbrunner** [comment, marked as answer]
> Is there a way to reconfigure the plugin (e.g. change Server) without restarting strongswan?

No, currently not.
